modules/installation/proc-configuring-an-rhdh-instance-with-tls-in-kubernetes.adoc

```diff
@@ -0,0 +1,104 @@
+[id="proc-configuring-an-rhdh-instance-with-tls-in-kubernetes_{context}"]
+= Configuring an {product-very-short} instance with a TLS connection in Kubernetes
+
+You can configure an {product-very-short} instance with a Transport Layer Security (TLS) connection in your Kubernetes cluster, such as an Azure Red Hat OpenShift (ARO) cluster. You can also use a public Certificate Authority (CA)-signed certificate to secure your {product-very-short} instance.
```

Comment:
Small note: my understanding is that you can **only** target a cluster with a public ca-signed certificate (such as ARO or other cloud providers or your own with proper configuration), as self-signed certificates are not supported in this case.

Reply:
It is a madatory thing to have a cluster configured with public Certificate Authority (CA)-signed certificate.

Modified the text:

You can configure an {product-very-short} instance with a Transport Layer Security (TLS) connection in a Kubernetes cluster, such as an Azure Red Hat OpenShift (ARO) cluster, any other cloud provider cluster, or your own cluster with proper configuration. However, You must use a public Certificate Authority (CA)-signed certificate to configure your Kubernetes cluster.